sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -153,6 +187,19 @@ def _get_active_connections(self, cursor):
         self.log.debug("loaded sql server current connections len(rows)=%s", len(rows))
         return rows
 
+    @tracked_method(agent_check_getter=agent_check_getter, track_result_length=True)
+    def _get_idle_blocking_sessions(self, cursor, blocking_session_ids):
+        # The IDLE_BLOCKING_SESSIONS_QUERY contains minimum information on idle blocker
+        self.log.debug("collecting sql server idle blocking sessions")
```

Comment:
That debug statement might not provide much additional info than the one below where we log the query that we're running. I've vote for removing it.